packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -142,11 +142,25 @@ class Directionality extends InheritedWidget {
 /// This is more efficient than adding and removing the child widget from the
 /// tree on demand.
 ///
+/// ## Opacity Animation
+///
+/// `Opacity` animations should be built using [AnimatedOpacity] rather than
+/// manually updating the `Opacity` Widget.
+///
+/// The 'Opacity' Widget rebuilds its subtree every frame of an animation.
```

Comment:
This sentence doesn't make much sense. Opacity doesn't do anything in an animation. Maybe try something like `Animating an [Opacity] widget directly causes the widget (and possible its subtree) to rebuild each frame, which is not very efficient. Consider using an [AnimatedOpacity] instead. [AnimatedOpacity] uses...`